libyul/backends/evm/EVMDialect.cpp

```diff
@@ -109,6 +109,27 @@ pair<YulString, BuiltinFunctionForEVM> createFunction(
 	return {name, f};
 }
 
+set<YulString> createReservedIdentifiers(bool _objectAccess)
```

Comment:
Why not just `reservedIdentifiers`?

Reply:
Because the other helper is called `createBuiltins`. I can change it though.